### ISSUE TITLE
Support .net Core 3.0

### ISSUE THREAD

**Issue by baronfel** [humhei/FCSWatch#46]
.net core global tools have to opt-in to support for other major versions of the .net core runtime.  This can be done by compiling for `netcoreapp3.0` as another target framework (possibly difficult), or by adding a `runtimeconfig.template.json` to the same directory as your dotnet-global-tool project (see [FAKE](https://github.com/fsharp/FAKE/blob/release/next/src/app/fake-cli/runtimeconfig.template.json) for an example).

**Comment by isaacabraham** [comment]
@baronfel yes, and ideally latest version of F# compiler - looks like things like implicit yield aren't supported
